web/src/app/pages/one-push/one-push.component.html

```diff
@@ -6,6 +6,11 @@
     <app-button link='/{{ pushHandle | async }}'>{{ pushHandle | async }}</app-button>
 </app-button-row>
 
+<app-bar-chart *ngIf = 'pushInfos | async' 
+                [pushInfos]='pushInfos | async' 
+                [currentPush] = 'pushInfo | async'>
```

Comment:
```suggestion
<app-bar-chart *ngIf='pushInfos | async' 
                [pushInfos]='pushInfos | async' 
                [currentPush]='pushInfo | async'>
```

Typically we don't pus spaces around `=` for HTML properties.